apps/portals/src/portal-components/SynapseComponentCollapse.tsx

```diff
@@ -0,0 +1,72 @@
+import { SynapseConfig } from 'types/portal-config'
+import React, { useState } from 'react'
+import { Box, Collapse, SxProps, Typography } from '@mui/material'
+import {
+  KeyboardArrowDownTwoTone,
+  KeyboardArrowUpTwoTone,
+} from '@mui/icons-material'
+import { SynapseComponent } from '../SynapseComponent'
+
+export type SynapseComponentCollapseProps = {
+  text: string
+  synapseConfig: SynapseConfig
```

Comment:
Most of this file is styling.  Has state to know if it is collapsed or not, and uses a MUI Collapse and Sage SynapseComponent to do the heavy lifting